src/disco/components/Addon.js

```diff
@@ -286,6 +287,37 @@ export function mapDispatchToProps(dispatch, { _tracking = tracking,
   // eslint-disable-next-line no-param-reassign
   _dispatchEvent = _dispatchEvent || document.dispatchEvent;
 
+  function showInfo({ name, iconUrl, i18n }) {
+    if (_config.has('useUiTour') && _config.get('useUiTour')) {
+      _dispatchEvent(new CustomEvent('mozUITour', {
+        bubbles: true,
+        detail: {
+          action: 'showInfo',
+          data: {
+            target: 'appMenu',
+            icon: iconUrl,
+            title: i18n.gettext('Your add-on is ready'),
+            text: i18n.sprintf(
+              i18n.gettext('Now you can access %(name)s from the toolbar.'),
```

Comment:
Ooooohhhh okay yeah I should've realised.

Cool, all good, then. My r+wc is now just an r+ 😄